cmd/manager/main.go

```diff
@@ -146,12 +146,17 @@ func main() {
 
 // addMetrics will create the Services and Service Monitors to allow the operator export the metrics by using
 // the Prometheus operator
-func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
-	if err := serveCRMetrics(cfg); err != nil {
+func addMetrics(ctx context.Context, cfg *rest.Config) {
```

Comment:
@vebken-splunk Have you tried updating local operator-sdk version to 0.18.2 and running make generate?

And also should we update our documentation [here](https://github.com/splunk/splunk-operator#prerequisites)

Reply:
Thanks for the comment, working on trying that out. 

And yes I believe that documentation should be updated, will take care of that.

Reply:
I think this might be more of an issue, it seems like the CSV generation (https://v0-18-x.sdk.operatorframework.io/docs/olm-integration/generating-a-csv/)  in the newer versions of the Operator SDK runs into issues with the way we are appending older API versions to CRDs 
  ```
  - name: v1beta1
    served: true
    storage: false
  - name: v1alpha3
    served: true
    storage: false
  - name: v1alpha2
    served: true
    storage: false
```
It will now error out if we don't include a pkg/apis/enterprise/<api_version> directory for each supported version. 
```
FATA[0175] error generating CSV: error generating ClusterServiceVersion base metadata: failed to set CRD descriptors for enterprise.splunk.com/v1beta1, Kind=ClusterMaster: no package found for v1beta1
```
Looking into how to best update our code for this new behavior.

Reply:
Is this hitting https://github.com/operator-framework/operator-sdk/issues/3002 ?

Reply:
I saw this as well, I believe it is a separate issue than the CSV errors I have been observing.